### PR TITLE
Add notes from 100th meeting, 2017-07-20

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2017-07-20-startup.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+title: "HSF Weekly Meeting #100, July 20, 2017"
+layout: default
+---
+
+# HSF Weekly Meeting #100, July 13, 2017
```
Comment (hegner) [inline organization/_posts/2017-07-20-startup.md]: July 20 ;-)

Comment (graeme-a-stewart) [inline organization/_posts/2017-07-20-startup.md]: Fixed - not sure about the Travis failure as that's a URL we had for ages...

**Comment by jouvin** [comment]
@graeme-a-stewart Please, could you rebase on the last upstream/master? I just merged a change by Benedikt fixing the check failure. This is again a US site causing problems with html-proofer but as many people edit directly on the web, it doesn't go through the checks and we discover the problem with later unrelated changes...

**Comment by graeme-a-stewart** [comment]
Hi @jouvin - rebased. Thanks for fixing that URL issue.

**Pull request merged**